pull

```diff
@@ -86,241 +52,147 @@ while [ "$1" != "" ]; do
-    docker-compose exec -T database mysql --user="$dockerDatabaseUser" --password="$dockerDatabasePassword" $dockerDatabaseName < $databaseFilename
-  fi
-  if [ $environment = 'local' ];
-  then
-    mysql --user="$localDatabaseUser" --password="$localDatabasePassword" --host="$localDatabaseHost" $localDatabaseName < $databaseFilename
-  fi
-  echo ''
+# Export database
+headline '⬆️ Export database'
+if database_exists; then
+    remoteDatabaseName=$vvcode
+    remoteDatabaseUser=$vvcode
+    remoteDatabasePassword=$vvcode
+    remoteDatabaseHost=localhost
 
-  # Clean up database files
-  echo 'Clean up database files'
-  echo '-----------------------'
-  rm $databaseFilename
-  echo 'Local SQL file was removed.'
-  if [ $hoster = 'mittwald' ];
-  then
-    ssh -T $remoteServerUser@$remoteServerHost -p $remoteServerPort "
-    rm /home/www/$remoteServerUser/tmp/$databaseFilename
-    "
-  else
     ssh -T $remoteServerUser@$remoteServerHost -p $remoteServerPort "
-    rm /home/forge/$remoteServerHost/tmp/$databaseFilename
+        cd $serverPath/ && mkdir -p tmp
+        (docker compose exec database mysqldump --ignore-table=$remoteDatabaseName.be_sessions --ignore-table=$remoteDatabaseName.cache_md5params --ignore-table=$remoteDatabaseName.cache_treelist --ignore-table=$remoteDatabaseName.cache_hash --ignore-table=$remoteDatabaseName.cache_hash_tags --ignore-table=$remoteDatabaseName.cache_imagesizes --ignore-table=$remoteDatabaseName.cache_imagesizes_tags --ignore-table=$remoteDatabaseName.cache_pages --ignore-table=$remoteDatabaseName.cache_pages_tags --ignore-table=$remoteDatabaseName.cache_pagesection --ignore-table=$remoteDatabaseName.cache_pagesection_tags --ignore-table=$remoteDatabaseName.cache_rootline --ignore-table=$remoteDatabaseName.cache_rootline_tags --ignore-table=$remoteDatabaseName.cache_treelist --ignore-table=$remoteDatabaseName.cf_extbase_datamapfactory_datamap --ignore-table=$remoteDatabaseName.cf_extbase_datamapfactory_datamap_tags --ignore-table=$remoteDatabaseName.cf_extbase_object --ignore-table=$remoteDatabaseName.cf_extbase_object_tags --ignore-table=$remoteDatabaseName.cf_extbase_reflection --ignore-table=$remoteDatabaseName.cf_extbase_reflection_tags --ignore-table=$remoteDatabaseName.tx_extensionmanager_domain_model_extension --ignore-table=$remoteDatabaseName.sys_log --user="$remoteDatabaseUser" --password='$remoteDatabasePassword' --host="$remoteDatabaseHost" $remoteDatabaseName) > tmp/$databaseFilename
     "
-  fi
-  echo 'Remote SQL file was removed.'
-  echo ''
+    catch "Could not create a sql dump."
+    scp -P $remoteServerPort $remoteServerUser@$remoteServerHost:$serverPath'/tmp/'$databaseFilename ./
+    catch "Could not download database."
+
+    # Import database
+    headline '⬇️ Import database'
+    databaseUser=$vvcode
+    databaseName=$vvcode
+    databaseHost=$vvcode'_database'
+    databasePassword=$vvcode
+
+    docker cp $databaseFilename $databaseHost:/var/www/html/$databaseFilename
```

Comment:
The import failed in my test, because the container name is different. Maybe it would be less error-prone to remove the vvcode here and use the `docker-compose cp` command?

```suggestion
    databaseHost='database'
    databasePassword=$vvcode

    docker-compose cp $databaseFilename $databaseHost:/var/www/html/$databaseFilename
```